.gitignore

```diff
@@ -0,0 +1 @@
+.vs/
```

Comment:
Ensure you added the correct .gitignore file by not committing bin and obj folders.